src/hdmf/term_set.py

```diff
@@ -94,3 +115,39 @@ def __getitem__(self, term):
         except KeyError:
             msg = 'Term not in schema'
             raise ValueError(msg)
+
+    def schemasheets_convert(self):
+        try:
+            import yaml
+            from linkml_runtime.utils.schema_as_dict import schema_as_dict
+            from schemasheets.schemamaker import SchemaMaker
+        except ImportError:
+            msg="Install schemasheets."
+            raise ValueError(msg)
+        schema_maker = SchemaMaker()
+        tsv_file_paths = glob.glob(self.schemasheets_folder + "/*.tsv")
+        schema = schema_maker.create_schema(tsv_file_paths)
+        schema_dict = schema_as_dict(schema)
+        schemasheet_schema_path = os.path.join(self.schemasheets_folder, f"{schema_dict['name']}.yaml")
+
+        with open(schemasheet_schema_path, "w") as f:
+            yaml.dump(schema_dict, f)
+
+        return schemasheet_schema_path
+
+    def enum_expander(self):
+        try:
+            warnings.filterwarnings("ignore", category=DeprecationWarning)
+            from oaklib.utilities.subsets.value_set_expander import ValueSetExpander
+        except ImportError:
+            msg = 'Install oaklib.'
+            raise ValueError(msg)
+        expander = ValueSetExpander()
+        # TODO: linkml should raise a warning if the schema does not have dynamic enums
+        enum = list(self.view.all_enums())
+        schema_dir = os.path.dirname(self.term_schema_path)
+        file_name = os.path.basename(self.term_schema_path)
+        output_path = os.path.join(schema_dir, f"expanded_{file_name}")
```

Comment:
Can you clarify why the term set is being expanded in the same place and why a user should not provide a path? If either option is fine, then I would add `output_path = None`  as an input argument and then set the output_path to be in place if the user does not provide a path.

Reply:
This is an internal method. In the spirit of minimizing fields the user has to specify, the new schema will be saved in the directory of the original schema (which is why I don't allow an output path).